### PR TITLE
fix: roadmap update

### PR DESCRIPTION
**Pull request by vinckr**
closes https://github.com/ory-corp/cloud/issues/996

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [x] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

**Comment by vinckr** [comment]
What do you think of "What comes next?", could also be "Upcoming features", or simply "Roadmap"

**Review by jfcurran — APPROVED** [review]
nice 🚀

**Comment by jfcurran** [comment]
'Roadmap' is good

File: src/page-content/content-pricing.tsx [hunk 1]
```diff
@@ -98,32 +100,30 @@ export const featurelist: FeaturelistProps = {
...
+          to={'/docs/early-access/'}
+          iconRight={ArrowRight}
+        >
+          Visit documentation
+        </Button>
+      ),
       features: [
         {
-          title: 'Advanced Identity and Access Management',
+          title: 'Advanced IAM',
```
Comment (piotrmsc) [inline src/page-content/content-pricing.tsx:126]: Let's leave
```suggestion
          title: 'Advanced Identity and Access Management',
```

**Pull request merged**